src/pages/CropSelector/CropCalendarView/RenderCrops.js

```diff
@@ -69,10 +67,8 @@ const RenderCrops = ({
         });
       }
     } else {
-      dispatch({
-        type: 'MY_CROP_LIST_LOCATION',
-        data: { from: 'selector' },
-      });
+      // dispatchRedux(myCropListLocation({ from: 'explorer' }));
```

Comment:
I think this should be uncommented and change the from property from 'explorer' to 'selector'.